### PR TITLE
Rename l7ProtocolName to appProtocol

### PR DESCRIPTION
**Pull request by tushartathgur**
Keeping the more standardized name for Layer 7 Protocol field

### CODE REVIEW

File: pkg/registry/registry_antrea.csv [hunk 1]
```diff
@@ -54,5 +54,5 @@ ElementID,Name,Abstract Data Type,Data Type Semantics,Status,Description,Units,R
 152,flowEndSecondsFromDestinationNode,dateTimeSeconds,,current,,,,,,,,56506,
 153,egressName,string,,current,,,,,,,,56506,
 154,egressIP,string,,current,,,,,,,,56506,
-155,l7ProtocolName,string,,current,,,,,,,,56506,
+155,appProtocol,string,,current,,,,,,,,56506,
```
Comment (antoninbas) [inline pkg/registry/registry_antrea.csv:57]: what about `appProtocolName`? I see that in standard IANA IE names, the `Name` suffix can be used explicitly to differentiate between string names and string ids (`vrfName`, `observationDomainName`, `observationDomainId`, ...)

Comment (tushartathgur) [reply pkg/registry/registry_antrea.csv:57]: appProtocol is more streamlined with https://github.com/kubernetes/api/blob/fe172d7dd4b6501923938705cfb69ee0b907c32d/core/v1/types.go#L5058 
may be we can use appProtocolID when we will introduce application layer protocol ID in the infix tables?

Comment (antoninbas) [reply pkg/registry/registry_antrea.csv:57]: I don't know if the K8s api is a good reference for this. It tends to use names exclusively, whereas IPFIX is a mix of names & ids

Comment (tushartathgur) [reply pkg/registry/registry_antrea.csv]: changed!

**Review by antoninbas — APPROVED** [review]

**Pull request merged**